docs/input_csv.md

```diff
@@ -0,0 +1,29 @@
+# Input CSV
+To build a template, provide a CSV with information about the images you'd like to use. Each line in the CSV represents a stack of images.
+
+### Required Columns
+- `subject_id`: Unique identifier for the subject. Do not use spaces, hyphens (`-`) or underscores (`_`).
+- `source_filepath`: Full path to the source image stack
+- `resolution_z`: Voxel resolution in the Z axis (in μm)
+- `resolution_y`: Voxel resolution in the Y axis (in μm)
+- `resolution_x`: Voxel resolution in the X axis (in μm)
+- `origin`: Three letter describing the anatomical orientation of the image (e.g., `PSL`, `LSP`, `RAS`)
```

Comment:
Maybe link to the `brainglobe` docs page that describes these anatomical orientations? (for those who may not be familiar)